### DISCUSSION TITLE
Holiday availability

### DISCUSSION THREAD

**Discussion by jakaplan** [trilemma-dev/SecureXPC#35] [Announcements]
My availability over the next few weeks (all dates NZDT):
- 18 Dec: unavailable
- 19 Dec: possibly limited availability in the evening
- 20-21 Dec: available
- 22 Dec: possibly limited availability during the day
- 23 Dec - 4 Jan: unavailable (and often completely without any cell service)

**Comment by jakaplan** [comment]
@amomchilov FYI

**Reply by amomchilov** [reply]
Thanks for letting me know. Here's mine (all times EST):

Dec 23: I'm moving into our new house! :D
Dec 23 - Jan 4: Free in principle, but mostly just moving and setting up shop. I'll be available to respond to messages, at least.

You keep a pretty low profile on the internet from what I can tell, so I suspect you won't be interested, but how do you feel about making a discord or slack group for these projects?

**Reply by jakaplan** [reply]
> You keep a pretty low profile on the internet from what I can tell

Indeed.

> I suspect you won't be interested, but how do you feel about making a discord or slack group for these projects?

Not something I'd want to do at this time, but open to revisiting this in the future. I'm not very "real time" in most of my Internet communication, so I think a Slack or Discord group with me as one of the primary participants is likely to disappoint.

**Reply by amomchilov** [reply]
Haha fair enough!

I think this "in the open" development is good for OSS anyway, even if it does mean that sometimes it takes a bit longer to clarify things (18 hours of time difference would do that, lol).
